src/eXpansion/Framework/GameShootmania/DependencyInjection/eXpansionGameShootmaniaExtension.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace eXpansion\Framework\GameShootmania\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+class eXpansionGameShootmaniaExtension extends Extension
+{
+    /**
+     * Loads a specific configuration.
+     *
+     * @param array            $configs   An array of configuration values
+     * @param ContainerBuilder $container A ContainerBuilder instance
+     *
+     * @throws \InvalidArgumentException When provided tag is not defined in this extension
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('data_providers.yml');
+        $loader->load('script_methods.yml');
```

Comment:
Remove as the script_methods.yml file is empty.

Reply:
i thought i keep it for future, but can do.